package.json

```diff
@@ -5,28 +5,32 @@
   "main": "index.js",
   "scripts": {
     "eslint": "eslint .",
-    "test": "npm run eslint"
+    "test": "npm run eslint",
+    "prettier": "prettier --config .prettierrc '**/*.{js,json}'"
   },
   "repository": "blogfoster/eslint-config-blogfoster",
-  "keywords": [
-    "eslint"
-  ],
+  "keywords": ["eslint"],
   "author": "blogfoster GmbH",
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/blogfoster/eslint-config-blogfoster/issues"
   },
   "homepage": "https://github.com/blogfoster/eslint-config-blogfoster#readme",
   "dependencies": {
-    "babel-eslint": "^7.2.3",
-    "eslint-config-airbnb": "^15.0.1",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-mocha": "^4.9.0",
-    "eslint-plugin-react": "^7.1.0",
-    "estraverse-fb": "^1.3.1"
```

Comment:
why did we remove this `estraverse-fb`, is it not needed anymore?

Reply:
not needed anymore

Reply:
ok